gui/app/main.cpp

```diff
@@ -80,31 +81,53 @@ int main(int argc, char *argv[]) {
     device = std::make_unique<RespiraConnectedDevice>(
         parser.value(serialPortOption));
   } else {
+    QFile file(":/sample-data/2020-05-14-pip15-peep5-rr12-ie23.csv");
```

Comment:
Heh ok.

Maybe put in a comment that making this generic isn't just a matter of making the filename configurable, because you're also assuming that the peep is 5?